test/e2e/azure_test.go

```diff
@@ -589,6 +589,16 @@ var _ = Describe("Workload cluster creation", func() {
 				},
 			}, result)
 
+			By("Validating that webhooks are working as expected", func() {
+				AKSWebhooksSpec(ctx, func() AKSWebhooksSpecInput {
+					return AKSWebhooksSpecInput{
+						Cluster:       result.Cluster,
+						MachinePools:  result.MachinePools,
+						WaitIntervals: e2eConfig.GetIntervals(specName, "wait-machine-pool-nodes"),
+					}
+				})
+			})
+
 			By("Upgrading the Kubernetes version of the cluster", func() {
```

Comment:
Do we want to keep the upgrade test running first immediately after the cluster is created?

Reply:
I don't think I care, I put these here to get faster test cycles.